File: olo-promotions-sdk-csharp/Olo.Promotions.SDK/Responses/VoidRedemptionResponse.cs
using Olo.Promotions.SDK.Responses.Models;

namespace Olo.Promotions.SDK.Responses
{
    /// <summary>
    /// Returned for a successful call to the Void Redemption endpoint.
    /// </summary>
    public class VoidRedemptionResponse
    {
        public Transaction Transaction { get; set; }
    }
}

File: olo-promotions-sdk-go/pkg/promotions/models/responses/promotions/void_redemption_response.go
package promotions

import "olo-promotions-sdk-go/pkg/promotions/models/responses"

// VoidRedemptionResponse is returned for a successful call to the Void Redemption endpoint.
type VoidRedemptionResponse struct {
	// Transaction contains the transaction details for the voided redemption.
	Transaction responses.Transaction `json:"transaction"`
}
